.github/workflows/charm-release.yaml

```
@@ -78,10 +78,16 @@ jobs:
           sudo snap install charmcraft --classic --channel latest/stable
           cd $GITHUB_WORKSPACE/charm/${{ inputs.charm-path }}
           # Get the libs folder name
-          libraries_folder=$(yq .name metadata.yaml | tr - _)
-          # For each library belonging to the charm, publish it
-          for lib in $(find lib/charms/$libraries_folder -type f | sed 's|lib/||' | sed 's/.py//' | sed 's|/|.|g'); do
-            charmcraft publish-lib $lib
-          done
+          if [ -e metadata.yaml ]; then
```

```suggestion
          if [ -f metadata.yaml ]; then
```

`-f` also makes sure **metadata.yaml** is a regular file, I'd say it's abit more robust